src/index.spec.ts

```diff
@@ -70,6 +70,7 @@ for (const service of [
         title: "Birthday party",
         start: "2019-12-29",
         allDay: true,
+        status: "CONFIRMED",
```

Comment:
I could add a new test just for this if you'd like? I would probably break out a new `describe` that is ics-specific, since status doesn't apply to the other services